### PR TITLE
Tables API: small nits

### PR DESCRIPTION
**Pull request by spolu**
## Description

Improve errors from the tables API as encountered while documenting it.

## Risk

N/A

## Deploy Plan

- deploy `front`

### CODE REVIEW

File: front/pages/api/v1/w/[wId]/data_sources/[name]/tables/index.ts [hunk 1]
```diff
@@ -98,7 +99,8 @@ async function handler(
           status_code: 500,
           api_error: {
             type: "internal_server_error",
-            message: "Failed to get tables.",
+            message: "Failed to retrieve tables.",
+            data_source_error: tablesRes.error,
```
Comment (fontanierh) [inline front/pages/api/v1/w/[wId]/data_sources/[name]/tables/index.ts:103]: is `data_source_error` the right name here ?

**Review by fontanierh — APPROVED** [review]
thx

File: front/pages/api/v1/w/[wId]/data_sources/[name]/tables/index.ts (hunk 1 above)
Comment (spolu) [reply front/pages/api/v1/w/[wId]/data_sources/[name]/tables/index.ts:103]: We're still in the context of data sources so I think that's fine that's just a way to surface more granular errors.

We support `data_source_error` `run_error` `app_error` and `connectors_error` as such additional error types

Comment (fontanierh) [reply front/pages/api/v1/w/[wId]/data_sources/[name]/tables/index.ts:103]: Fair 👍

**Pull request merged**